### ISSUE TITLE
REPO ABANDONED - NEW LOCATION

### ISSUE THREAD

**Issue by stylesuxx** [mathiasvr/bluejay#59]
This repository is abandoned, development continues in the [new repository](https://github.com/bird-sanctuary/bluejay).

**Comment by swilkens** [comment]
@mathiasvr ?

**Comment by stylesuxx** [comment]
You can read more about what's going on in the [new orgs readme](https://github.com/bird-sanctuary). I am afraid you might wait a while for Mathias to answer.

**Comment by PhilippMolitor** [comment]
you should really update the readme with a big notice to make this clear, I wasn't aware of it until i looked at the issues list.

**Comment by stylesuxx** [comment]
@PhilippMolitor yeah - that's difficult to do if the repo is **ABANDONED** by the owner.

**Comment by PhilippMolitor** [comment]
I wasn't aware mathias was the only one with full access, I understand :)

**Comment by stylesuxx** [comment]
Unfortunately, yes - otherwise we would have loved to "take it over" properly. In the new org, 4 peoples are owners, so we should be on the safe side...
